### DISCUSSION TITLE
Anyone have a video tutorial installing Oqtane on local machine?

### DISCUSSION THREAD

**Discussion by giffordwatkins** [oqtane/oqtane.framework#786] [Q&A]
I've been going over and over https://www.oqtane.org/Resources/Blog/PostId/542/installing-oqtane-on-iis  and I'm a little unclear on Step 8 - Add a Host entry  (am I adding a new file? or adding to one of the existing files in C:\Windows\System32\drivers\etc ?  (I tried adding "127.0.0.1 oqtane.dev" to existing "hosts" file, I tried creating a file called "oqtane.dev" and adding "127.0.0.1 oqtane.dev".

Some people think I'm smart, but I'm not all that smart... I see a module dev series here:  https://www.youtube.com/channel/UCrHoLk244v1Fo-nXeinleDg/videos  

If someone has a video tutorial of best practices installing Oqtane (on local or server), I'd be grateful. If video tutorials don't exist, then **_perhaps someone would like to do a Zoom session, record the steps, and then we'll have a smokin resource_** for the community!

Hey! I'm brand new to the Oqtane ecosystem and thought I'd dive in and get up a local instance so I could show my coders. We've been investing in the Laravel project... but hey; we might want to jump ship if I can provide a convincing case for Oqtane. 

First steps for me... is this the community for us.

**Comment by leigh-pointer** [comment]
Adding a host in IIS is done through site bindings.
![step2-site-bindings](https://user-images.githubusercontent.com/9447612/95347684-7635a380-08bd-11eb-8d0b-6b1eff639e3c.png)

**Reply by sbwalker** [reply]
@giffordwatkins you should add "127.0.0.1 oqtane.dev" to existing "hosts" file. Then if you ping oqtane.dev it should respond with 127.0.0.1. @leigh-pointer is correct about how to add the host name to the IIS site ( described in Step 7 in the blog )

**Reply by sbwalker** [reply]
I updated the blog
